build/BUILD

```diff
@@ -63,10 +63,6 @@ DOCKERIZED_BINARIES = {
         "base": ":busybox-libc",
         "target": "//plugin/cmd/kube-scheduler:kube-scheduler",
     },
-    "kube-aggregator": {
-        "base": ":busybox-libc",
-        "target": "//vendor/k8s.io/kube-aggregator:kube-aggregator",
```

Comment:
`//vendor/k8s.io/kube-aggregator:...` refers to the BUILD file in that directory (which does not exist). `//vendor:k8s.io/kube-aggregator/kube-aggregator` in contrast is the aggregator target in ´vendor/BUILD`.

There is a pending PR for gazel to create BUILD files also inside vendored dirs. Until then everything is flat in `vendor/BUILD`.